wsagent/che-wsagent-core/src/main/java/org/eclipse/che/wsagent/server/WsAgentModule.java

```diff
@@ -28,6 +32,14 @@
   protected void configure() {
     bind(ApiInfoService.class);
     bind(LivenessProbeService.class);
+    bind(ExecutorService.class)
+        .annotatedWith(Names.named("che.core.jsonrpc.major_executor"))
```

Comment:
I think it is better to make `che.core.jsonrpc.major_executor` it as constant of `CheWebSocketEndpoint`  class and also rename it to agent specific. It is not major nor minor, it's just the only agent Enpoint, right?